packages/grafana-plugin-configs/webpack.config.ts

```diff
@@ -158,7 +190,30 @@ const config = async (env: Record<string, unknown>): Promise<Configuration> => {
       uniqueName: pluginJson.id,
     },
 
+    optimization: {
+      minimize: Boolean(env.production),
+      minimizer: [
+        new TerserPlugin({
+          terserOptions: {
+            format: {
+              comments: (_, { type, value }) => type === 'comment2' && value.trim().startsWith('[create-plugin]'),
+            },
+            compress: {
+              drop_console: ['log', 'info'],
+            },
+          },
+        }),
+      ],
+    },
+
     plugins: [
+      virtualPublicPath,
+      // Insert create plugin version information into the bundle so Grafana will load from cdn with script tags.
+      new BannerPlugin({
+        banner: '/* [create-plugin] version: 5.22.0 */',
```

Comment:
Is this the minimum supported version for loading from the CDN?

Reply:
No, I believe the minimum is 4.15.0 but this was the version in create-plugin package.json when I copied these changes across.